invenio_audit_logs/services/config.py

```diff
@@ -39,17 +38,12 @@ class AuditLogSearchOptions(SearchOptionsBase):
     sort_default = "newest"
     sort_default_no_query = "newest"
 
-    sort_direction_default = "asc"
-    sort_direction_options = {
-        "asc": dict(title=_("Ascending"), fn=asc),
-        "desc": dict(title=_("Descending"), fn=desc),
-    }
-
     query_parser_cls = QueryParser.factory(
         fields=[
             "id",
             "action",
             "user.id",
+            "user.name",  # TODO: Resolve to id on search?
```

Comment:
I would avoid the TODOs: by experience, they are never fixed.
Is it a blocker? Then we need to fix it.
Is it a nice to have? I would not put anything.

Reply:
Discussed IRL, nothing to fix, would be nice to have name to id resolving, to fetch all records of that user by name in the future because username can change